iati/default.py

```diff
@@ -16,27 +16,29 @@
 import iati.resources
 
 
-def get_default_version_if_none(version):
-    """Return the default version number if the input version is None. Otherwise returns the input version as is.
+def _specific_version_for(version):
+    """Convert a version number into the most appropriate Decimal Version.
+
+    * Decimal Versions will remain unchanged.
+    * Integer Versions will return the latest Decimal Version within the Integer.
 
     Args:
-        version (str or None): The version to test against.
+        version (str): The version to obtain a specific version for.
 
     Raises:
-        ValueError: When a specified version is not a valid version of the IATI Standard.
+        ValueError: When a specified version cannot be converted to a valid version of the IATI Standard.
 
     Returns:
-        str: The default version if the input version is None. Otherwise returns the input version.
-
-    Note:
-        The default version of the Standard is deemed to be the latest version.
+        str: The Decimal Version of the Standard that the input version relates to.
 
     """
-    if version is None:
-        return iati.constants.STANDARD_VERSION_LATEST
-    elif version not in iati.constants.STANDARD_VERSIONS:
-        raise ValueError("Version {0} is not a valid version of the IATI Standard.".format(version))
-    return version
+    if version in iati.constants.STANDARD_VERSIONS:
+        return version
+    elif version in [str(v) for v in iati.constants.STANDARD_VERSIONS_MAJOR]:
+        decimals_for_integer = [v for v in iati.constants.STANDARD_VERSIONS if v[0] == version]
```

Comment:
Won't work when we get beyond version 9, but maybe we shouldn't worry about that??!  Could add a todo? Could use `v.split('.')[0] == version`??

Reply:
a1d0a449d6a3507722a1336863743f0703a20eef

:P